web/src/app/workplace/tasklist/tasklist.component.html

```diff
@@ -6,7 +6,7 @@
   <div *ngIf="!requestInProgress">
     <ul #taskList id="task-list-container" class="list-group">
       <li class="list-group-item" *ngIf="tasks === undefined || tasks.length === 0" type="text">
-        <b>This Workbasket has no Tasks</b>
```

Comment:
Text is ok, but add an icon like workbasket list

Reply:
Not only that. It has to be a different container since the "This workbasket has no Tasks" message should still be there :)